src/main/java/uk/ac/ebi/ampt2d/accession/file/UuidFileAccessioningService.java

```diff
@@ -28,14 +30,14 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
-//@Service
-//@Profile("file-uuid")
+@Service
 public class UuidFileAccessioningService extends AccessioningService<UuidFile, UUID> {
 
+    @Autowired
     private UuidFileAccessionRepository fileRepository;
 
-    public UuidFileAccessioningService(String namespace) {
-        super(new UuidAccessionGenerator<>(namespace));
+    public UuidFileAccessioningService(@Autowired AccessioningProperties properties) {
```

Comment:
Not sure, but right now Spring does automatic constructor autowiring without annotation at least for Beans, Services should work too.